### PR TITLE
Using newrelic

### PR DESCRIPTION
**Pull request by e0d**

### CODE REVIEW

File: playbooks/roles/xqwatcher/templates/edx/app/supervisor/conf.d/xqwatcher.conf.j2 [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+
+{% if COMMON_ENABLE_NEWRELIC %}
+{% set executable = xqwatcher_venv_dir + '/bin/newrelic-admin run-program ' + xqwatcher_venv_dir + '/bin/python' %}
+{% else %}
+{% set executable = xqwatcher_venv_dir + '/bin/python' %}
+{% endif %}
+
 [program:xqwatcher]
-command={{ xqwatcher_venv_dir }}/bin/python -m {{ xqwatcher_module }} -d ../conf.d/
+command={{ executable }} -m {{ xqwatcher_module }} -d ../conf.d/
```
Comment (feanil) [inline playbooks/roles/xqwatcher/templates/edx/app/supervisor/conf.d/xqwatcher.conf.j2]: should this point to `{{ xqueue_app_dir }}` now that xqueue watcher reads a toplevel conf along with the content of conf.d?

File: playbooks/roles/xqwatcher/templates/edx/app/supervisor/conf.d/xqwatcher.conf.j2 [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 [program:xqwatcher]
-command={{ xqwatcher_venv_dir }}/bin/python -m {{ xqwatcher_module }} -d ../conf.d/
+command={{ executable }} -m {{ xqwatcher_module }} -d {{ xqueue_app_dir }}
 process_name=%(program_name)s
 directory={{ xqwatcher_code_dir }}
 stdout_logfile={{ xqwatcher_supervisor_log_dir }}/%(program_name)-stdout.log
 stderr_logfile={{ xqwatcher_supervisor_log_dir }}/%(program_name)-stderr.log
 killasgroup=true
 stopasgroup=true
+a
```
Comment (feanil) [inline playbooks/roles/xqwatcher/templates/edx/app/supervisor/conf.d/xqwatcher.conf.j2]: typo?

**Comment by feanil** [comment]
Other than the typo looks good.

**Comment by jarv** [comment]
nit: might be able ot use `{{ xqwatcher_venv_bin }}` instead, I did that for the xqueue role.

**Pull request merged**